build.gradle

```
@@ -37,6 +38,11 @@ buildscript {
     runtime: "androidx.compose.runtime:runtime:${versions.composeUi}",
     foundation: "androidx.compose.foundation:foundation:${versions.composeUi}",
     material: "androidx.compose.material:material:${versions.composeUi}",
+    composeUiToolingPreview: "androidx.compose.ui:ui-tooling-preview:${versions.composeUi}",
+    uiTooling: "androidx.compose.ui:ui-tooling:${versions.composeUi}",
+    paparazzi: "app.cash.paparazzi:paparazzi:${versions.paparazzi}",
+    paparazziPlugin: "app.cash.paparazzi:paparazzi-gradle-plugin:${versions.paparazzi}",
+    layoutlib: 'app.cash.paparazzi:layoutlib-native-jdk11:2022.1.1-canary-f5f9f71',
```

```suggestion
    layoutlib: 'app.cash.paparazzi:layoutlib-native-jdk11:2021.2.1-patch1-fa3aa65',
```

this is the latest released/supported version in Paparazzi